app/main.py

```diff
@@ -16,4 +16,15 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+sorted_variables = {
+    "mutable": [
+        my_favourite_films,
+        marks,
+        collection_of_coins],
```

Comment:
It is possible but:
- Opening and closing square brackets are not in the same style
- This value is not too long, you can write it in a row